python/python/bystro/ancestry/ancestry_types.py

```diff
@@ -0,0 +1,179 @@
+"""Classes for common shapes of data in ancestry."""
+import re
+from collections import Counter
+from collections.abc import Sequence
+from typing import TypeVar
+
+import attrs
+from attrs import field
```

Comment:
Since you're importing attrs here but use attrs.field, maybe importing field could be skipped here? And then uses of field converted to attrs.field for consistency? Will mark uses of field without attrs below.